src/personen/personen.lambda.ts

```diff
@@ -80,13 +98,23 @@ export async function callHaalCentraal(content: string, personenSecrets: Persone
       },
     );
 
-    const data = await resp.json();
-
-    return {
-      statusCode: resp.status,
-      body: JSON.stringify(data),
-      headers: { 'Content-Type': 'application/json' },
-    };
+    var data: any;
+    const contentType = resp.headers.get('content-type');
+    if (contentType && contentType.includes('application/json')) {
+      data = await resp.json();
+      return {
+        statusCode: resp.status,
+        body: JSON.stringify(data),
+        headers: { 'Content-Type': 'application/json' },
+      };
+    } else {
+      data = await resp.text();
+      return {
+        statusCode: resp.status,
+        body: data,
+        headers: { 'Content-Type': 'text/plain' },
```

Comment:
Weet je zeker dat het altijd text moet zijn als het niet json is?

Reply:
verwijderd